miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -20,6 +20,11 @@ abstract class MiniAppMessageBridge {
         callback: (isGranted: Boolean) -> Unit
     )
 
+    /** This host app info can apply to analytics, app traffic, ads, etc.
+     *  It should be a meaningful keyword such as host app name to differentiate other host apps.
+     **/
+    abstract fun getHostAppInfo(): String
```

Comment:
I'm not sure if `MiniAppMessageBridge` is the right place for this. This class is intended for sending data between the host app and mini app, but the `hostAppInfo` isn't accessible from the Mini App.

I was thinking, maybe this could instead be an optional setting in the manifest meta-data? If the host app doesn't use the setting, then the SDK would automatically retrieve the App name using `ApplicationInfo`. What do you think @minh-rakuten @ts-nitish-a-jain ?

Reply:
I'm on it.

Reply:
Correct. 
In slack thread also, it was aligned, a few days back I think, that this needs to be a build time setting.

When not provided, this should be default value and no changes.
When provided, appended to the userAgentString.